lambda/app/src/routes.ts

```diff
@@ -487,4 +490,34 @@ export const setRoutes = (app: any) => {
       handleError(res, err);
     }
   });
+
+  app.post(`${BASE_PATH}/teams/:id/service-accounts`, async (req, res) => {
```

Comment:
Maybe a naive question, but are we not able to modify or delete service accounts once created?

Reply:
they might update data in the future, but for now we don't have the business requirements.

Reply:
sounds good